provisioner.go

```diff
@@ -330,17 +350,27 @@ type Disembedder func(src string) ([]byte, error)
 // NewDisembedderProvisioner creates a provisioner that can provision based on embedded data
 func NewDisembedderProvisioner(d Disembedder, pathAstilectron, pathElectron string, l astikit.StdLogger) Provisioner {
 	dp := &defaultProvisioner{l: astikit.AdaptStdLogger(l)}
-	dp.moverAstilectron = func(ctx context.Context, p Paths) (err error) {
+	dp.moverAstilectron = func(ctx context.Context, p Paths) (closeFunc func() error, err error) {
 		if err = Disembed(ctx, dp.l, d, pathAstilectron, p.AstilectronDownloadDst()); err != nil {
-			return fmt.Errorf("disembedding %s into %s failed: %w", pathAstilectron, p.AstilectronDownloadDst(), err)
+			return nil, fmt.Errorf("disembedding %s into %s failed: %w", pathAstilectron, p.AstilectronDownloadDst(), err)
 		}
-		return
+		return func() (err error) {
+			if err = os.Remove(p.AstilectronDownloadDst()); err != nil {
+				err = fmt.Errorf("removing %s failed: %w", p.AstilectronDownloadDst(), err)
```

Comment:
you need to `return` here as well

Reply:
done. https://github.com/asticode/go-astilectron/pull/358/commits/4bac7b9d74646f5eec8c4f79dee7a2754e9d241c#diff-752145c98ffd12d3a2b292517f8cf139ca3bcaa6512dc88dd7d8628372fd5df4R362